assets/js/googlesitekit/data/create-notifications-store.js

```diff
@@ -259,10 +263,23 @@ export const createNotificationsStore = ( type, identifier, datapoint, {
 
 	const resolvers = {
 		*getNotifications() {
-			yield actions.fetchNotifications();
+			try {
+				const notifications = yield actions.fetchNotifications();
+				return actions.receiveNotifications( notifications );
+			} catch ( err ) {
+				return actions.receiveNotificationsFailed();
+			}
```

Comment:
This block here is old code and shouldn't be there. It can actually result in an error because `receiveNotificationsFailed` does not exist. It should be reverted to what it was before (line 262).

Reply:
I have removed this block and reverted this back to yield actions.fetchNotifications();